src/VirtoCommerce.CatalogPersonalizationModule.Web/module.manifest

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<module>
+    <id>VirtoCommerce.CatalogPersonalization</id>
+    <version>3.0.0</version>
+    <version-tag>rc.3</version-tag>
+    <platformVersion>3.0.0</platformVersion>
+    <dependencies>
+        <dependency id="VirtoCommerce.Core" version="3.0.0-rc.3" />
+        <dependency id="VirtoCommerce.Customer" version="3.0.0-rc.3" />
+        <dependency id="VirtoCommerce.Catalog" version="3.0.0-rc.3" />
+    </dependencies>
```

Comment:
Probably we need search module in dependencies

Reply:
added